src/api/l_audio_source.c

```diff
@@ -51,6 +51,22 @@ static int l_lovrSourceSetVolume(lua_State* L) {
   return 0;
 }
 
+static int l_lovrSourceGetSpatial(lua_State* L) {
```

Comment:
Lua API functions use the `is` prefix instead of `get` when exposing a boolean (see `isLooping` and `isPlaying`).